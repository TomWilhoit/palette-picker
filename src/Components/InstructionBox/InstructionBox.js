import React from 'react';

export const InstructionBox = (props) => {
  return (
    <div className="instruction-box">
      {props.message}
    </div>
  )
}