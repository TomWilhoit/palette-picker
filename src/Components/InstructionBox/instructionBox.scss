.instruction-box {
  background-color: $container-bkgd;
  border-left: 5px solid $border;
  border-right: 5px solid $border;
  border-top: 2px solid $border;
  padding: 6px;
  text-align: left;
}