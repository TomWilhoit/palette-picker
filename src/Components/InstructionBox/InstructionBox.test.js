import React from "react";
import { InstructionBox } from "./InstructionBox";
import { shallow } from "enzyme";

describe("InstructionBox", () => {
  let wrapper;
  let mockInstructionMessage = "1. Always pass mock props when testing";
  beforeEach(() => {
    wrapper = shallow(<InstructionBox message={mockInstructionMessage} />);
  })

  it("should match the snapshot with all data passed in", () => {
    expect(wrapper).toMatchSnapshot();
  })
})