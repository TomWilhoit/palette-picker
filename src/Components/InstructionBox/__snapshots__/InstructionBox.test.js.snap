// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstructionBox should match the snapshot with all data passed in 1`] = `
<div
  className="instruction-box"
/>
`;
